src/classes/Warehouse.java

```diff
@@ -0,0 +1,51 @@
+package classes;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Warehouse {
+    private Map<String, Integer> stock;

```

Comment:
Non sarebbe meglio al posto di String usare direttamente product?

Reply:
quindi cambiare poi i parametri in Product product, ci sta, faccio subito

Reply:
Si, così tutti i parametri sono accessibili direttamente dalla warehouse. Io nel cartello ho fatto così ma non so se sia il metodo migliore.